<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="les1PushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>What is the Stock Market?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="les2PushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Safe and Volatile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="les3PushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buy Stocks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="les4PushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sell Stocks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="les5PushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stock Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="les6PushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>600</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Market Shorting</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>641</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Koding Stock Tutorial</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
